part-2-server/server.js

```diff
@@ -0,0 +1,30 @@
+var http = require("http");
+var fs = require("fs");
+
+var server = http.createServer(function(request, response){
+var url = request.url
+	
+    		
+	if(request.url === '/'){
+     	fs.readFile('index.html', function(err, data) {
+            response.write(data.toString());
+            response.end();
+        })
+
+    } else if (request.url === '/page-2' || request.url === '/page-3') {
+    	fs.readFile('page-2.html', function(err, data){
+    		response.write(data.toString());
+    		response.end();
+    	})
+
```

Comment:
Where is `i` coming from? We can't use `i` without first declaring it...would either need to declare this variable somewhere before this point, or move this within a for loop